test/Microsoft.AspNetCore.Mvc.Localization.Test/Internal/MvcLocalizationServiceCollectionExtensionsTest.cs

```diff
@@ -105,73 +61,22 @@ public void AddCustomLocalizers_BeforeAddLocalizationServices_AddsNeededServices
-                    Assert.Equal(ServiceLifetime.Singleton, service.Lifetime);
-                },
-                service =>
-                {
-                    Assert.Equal(typeof(IHtmlLocalizer<>), service.ServiceType);
-                    Assert.Equal(typeof(TestHtmlLocalizer<>), service.ImplementationType);
-                    Assert.Equal(ServiceLifetime.Transient, service.Lifetime);
-                },
-                service =>
-                {
-                    Assert.Equal(typeof(IViewLocalizer), service.ServiceType);
-                    Assert.Equal(typeof(TestViewLocalizer), service.ImplementationType);
-                    Assert.Equal(ServiceLifetime.Transient, service.Lifetime);
-                },
-                service =>
-                {
-                    Assert.Equal(typeof(HtmlEncoder), service.ServiceType);
-                    Assert.Same(testEncoder, service.ImplementationInstance);
-                }, service =>
-                {
-                    Assert.Equal(typeof(IOptions<>), service.ServiceType);
-                    Assert.Equal(typeof(OptionsManager<>), service.ImplementationType);
-                    Assert.Equal(ServiceLifetime.Singleton, service.Lifetime);
-                },
-                service =>
-                {
-                    Assert.Equal(typeof(IOptionsSnapshot<>), service.ServiceType);
-                    Assert.Equal(typeof(OptionsManager<>), service.ImplementationType);
-                    Assert.Equal(ServiceLifetime.Scoped, service.Lifetime);
-                },
-                service =>
-                {
-                    Assert.Equal(typeof(IOptionsMonitor<>), service.ServiceType);
-                    Assert.Equal(typeof(OptionsMonitor<>), service.ImplementationType);
-                    Assert.Equal(ServiceLifetime.Singleton, service.Lifetime);
-                },
-                service =>
-                {
-                    Assert.Equal(typeof(IOptionsFactory<>), service.ServiceType);
-                    Assert.Equal(typeof(OptionsFactory<>), service.ImplementationType);
-                    Assert.Equal(ServiceLifetime.Transient, service.Lifetime);
-                },
-                service =>
-                {
-                    Assert.Equal(typeof(IOptionsMonitorCache<>), service.ServiceType);
-                    Assert.Equal(typeof(OptionsCache<>), service.ImplementationType);
-                    Assert.Equal(ServiceLifetime.Singleton, service.Lifetime);
-                },
-                service =>
-                {
-                    Assert.Equal(typeof(IConfigureOptions<RazorViewEngineOptions>), service.ServiceType);
-                    Assert.Equal(ServiceLifetime.Singleton, service.Lifetime);
-                });
+            AssertContainsExpected(collection, expected);
         }
 
         [Fact]
         public void AddCustomLocalizers_AfterAddLocalizationServices_AddsNeededServices()
```

Comment:
Let's delete this test. It's not testing any sort of useful behavior (in reality, services that get added after will get resolved by the DI container).